.drone.star

```diff
@@ -1829,7 +1829,7 @@ def e2eTestsOnKeycloak(ctx):
                          "BASE_URL_OCIS": "ocis:9200",
                          "HEADLESS": "true",
                          "RETRY": "1",
-                         "REPORT_TRACING": "true",
+                         "REPORT_TRACING": "with-tracing" in ctx.build.title.lower(),
```

Comment:
sorry I don't get it. That means we only get tracing for keycloak test is we set `with-tracing` to the PR header?

Reply:
we get tracing if there's with-tracing in the title or tracings are enabled if a scenario is retried after failing.